pkgs/misc/emulators/retroarch/cores.nix

```diff
@@ -558,77 +437,47 @@ in with lib.licenses;
     makefile = "Makefile.libretro";
   };
 
-  mame2000 = mkLibRetroCore rec {
+  mame2000 = mkLibRetroCore {
     core = "mame2000";
-    src = fetchRetro {
-      repo = core + "-libretro";
-      rev = "e5d4a934c60adc6d42a3f87319312aad89595a15";
-      sha256 = "1zn63yqyrsnsk196v5f3nm7cx41mvsm3icpis1yxbma2r3dk3f89";
-    };
     description = "Port of MAME ~2000 to libretro";
-    license = gpl2Plus;
+    license = "MAME";
```

Comment:
I think it would be better to create the license then reference that variable here

Reply:
This is also how other cores not affected by this PR is using it: https://github.com/NixOS/nixpkgs/blob/master/pkgs/misc/emulators/retroarch/cores.nix#L808

Also, AFAIK `lib.licenses` only includes open-source licenses. So MAME license is a no-go.

Reply:
Correction: `lib.licenses` only includes licenses from SPDX, and MAME is not in SPDX list: https://spdx.org/licenses/